examples/component/sqldb-postgres-query/wadm.yaml

```diff
@@ -0,0 +1,60 @@
+---
+apiVersion: core.oam.dev/v1beta1
+kind: Application
+metadata:
+  name: go-sqldb-postgres-query
+  annotations:
+    description: |
+      Demo WebAssembly component using the wasmCloud SQLDB Postgres provider via the wasmcloud:postgres WIT interface
+    wasmcloud.dev/authors: wasmCloud team
+    wasmcloud.dev/source-url: https://github.com/wasmCloud/wasmCloud/blob/main/examples/golang/components/sqldb-postgres-query/wadm.yaml
+    wasmcloud.dev/readme-md-url: https://github.com/wasmCloud/wasmCloud/blob/main/examples/golang/components/sqldb-postgres-query/README.md
+    wasmcloud.dev/homepage: https://github.com/wasmCloud/wasmCloud/tree/main/examples/golang/components/sqldb-postgres-query
+    wasmcloud.dev/categories: |
+      database,sqldb,postgres,golang,example
+spec:
+  components:
+    - name: querier
+      type: component
+      properties:
+        image: file://./build/sqldb_postgres_query_s.wasm
+      traits:
+        # Govern the spread/scheduling of the component
+        - type: spreadscaler
+          properties:
+            instances: 1
+        # Establish a unidirectional link to the `sqldb-postgres` provider (the sqldb provider),
+        # so the `querier` component can make use of sqldb functionality provided Postgres
+        # (i.e. reading/writing to a database)
+        - type: link
+          properties:
+            target:
+              name: sqldb-postgres
+              config:
+                # NOTE: Normally this would be done via a secret, but for demo purposes we're it is
+                # hardcoded here
+                - name: pg-conf
+                  properties:
+                    POSTGRES_HOST: localhost
+                    POSTGRES_PORT: "5432"
+                    POSTGRES_USERNAME: postgres
+                    POSTGRES_PASSWORD: postgres
+                    POSTGRES_DATABASE: postgres
+                    POSTGRES_TLS_REQUIRED: "false"
+            namespace: wasmcloud
+            package: postgres
+            interfaces: [query]
+
+    # Add a capability provider that interacts with the filesystem
+    - name: sqldb-postgres
+      type: capability
+      properties:
+        image:
+          ghcr.io/wasmcloud/sqldb-postgres:0.7.1
```

Comment:
Can we bump this to the [latest version (`0.7.2`)](https://github.com/wasmCloud/wasmCloud/pkgs/container/sqldb-postgres)